packages/commonwealth/client/scripts/views/pages/overview/TopicSummaryRow.tsx

```diff
@@ -113,7 +112,7 @@ export const TopicSummaryRow = ({
                     >
                       {moment(getLastUpdated(thread)).format('l')}
                     </CWText>
-                    <NewThreadTag threadCreatedAt={thread.createdAt} />
+                    { isNewThread(thread.createdAt) && <CWTag label={'New'} type={'new'} iconName={'newStar'}/> }
```

Comment:
prettier: file is not formatted correctly

Reply:
Fixed!